ReadDiceKey.podspec

```diff
@@ -64,9 +65,13 @@ Pod::Spec.new do |s|
   # s.frameworks = 'UIKit', 'MapKit'
   # s.dependency 'AFNetworking', '~> 2.3'
 
-  s.dependency "OpenCV2", '~> 4.3.0'
+  s.dependency 'OpenCVXF'
   s.dependency 'Djinni', '~> 4.4.1'
   
+  s.pod_target_xcconfig = {
+      "HEADER_SEARCH_PATHS" => "\"/Users/bakhtiyor/Projects/UpWork/dicekeys/opencv/Headers\""
```

Comment:
This is a hard URL on your system.  Needs to be something that will work for others.